internal/prompter/prompter.go

```diff
@@ -14,7 +14,7 @@ import (
 //go:generate moq -rm -out prompter_mock.go . Prompter
 type Prompter interface {
 	Select(string, string, []string) (int, error)
-	MultiSelect(string, []string, []string) ([]int, error)
+	MultiSelect(prompt string, defaults []string, options []string) ([]int, error)
```

Comment:
Was this meant to be committed?

Reply:
yeah, I'm going to do the rest in my next PR